testing/helper.go

```diff
@@ -0,0 +1,25 @@
+package testing
```

Comment:
Let's try and avoid squatting the `testing` package name - I know they've done this in `indigo`, but I consider this bad practice since it'll conflict with Go's std `testing` package.

If this is just going to contain helpers, I'd name it `testenv`. If it's going to contain the actual integration tests, I'd name it `integration`. I can see an argument for either strategy, keeping integration tests in their own package ensures we test in a blackbox strategy and makes it easier to split the integration tests into their own GitHub Action job.  The other option keeps the tests closer to the code under test . I'd say for now, my gut feeling is to go with an `integration` package that contains any helpers and the "integration" suite.